Monika-After-Story/MonikaModDev
# Extend hours of night farewells

~This creates a specific rule for hours that allows us to check over multiple hour ranges as opposed to only one. This is especially useful if we want to have an event that we want available before and after midnight. Specifically, our 2 goodnight farewells right now both end at midnight, but they really should be available past midnight, potentially with different paths depending on what time it actually is (this and more to come later).~ Eventually to be paired with #2192 to create a better goodnight and good morning (or other wake-up time) experience.

This extends the `bye_going_to_sleep` and `bye_goodnight` farewells past midnight until 1am.

## Testing

- verify that using a normal goodbye between 8pm and midnight either results in `bye_going_to_sleep` or `bye_goodnight`

- verify that using a normal goodbye between midnight and 1am also results in the same 2 farewell options

- verify that using a normal goodbye outside of these time ranges does not result in these 2 farewell options
